styles.css

```diff
@@ -91,6 +95,331 @@ ul li a {
   font-family: Arial, sans-serif, 'Roboto';
 }
 
-.illustration {
-  display: none;
+.works {
+  display: grid;
+  grid-template-rows: auto auto auto auto auto auto auto auto;
+  row-gap: 10px;
+  justify-items: center;
+  margin: 0;
+}
+
+.works h2 {
+  color: #1c1a19;
+  font-family: Arial, sans-serif, 'Crete Round';
+  font-size: 40px;
+  font-weight: 400;
+  font-style: normal;
+  padding: 100px 24px 10px;
+}
+
+.works img {
+  padding: 25px;
+}
+
+.stories-intro {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.stories-intro img {
+  width: 100%;
+}
+
+.stories-intro h2 {
+  font-size: 32px;
+  font-weight: 400;
+  color: #091e42;
+  font-family: 'Crete Round', arial, sans-serif;
+  align-self: flex-start;
+  padding: 0 20px;
+}
+
+.stories-intro p {
+  font-weight: 400;
+  font-size: 16px;
+  color: #091e42;
+  font-family: arial, sans-serif, 'Inter';
+  line-height: 24px;
+  padding: 0 20px;
+}
+
+.pro-btn {
+  display: flex;
+  justify-content: flex-start;
+  gap: 12px;
+  padding: 5px 0;
+}
+
+.pro-btn .css-btn {
+  border: 1px solid #8993a4;
+  width: 49px;
+  height: 40px;
+  font-weight: 500;
+  font-family: arial, sans-serif, 'Inter';
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  cursor: pointer;
+}
+
+.pro-btn .html-btn {
+  border: 1px solid #8993a4;
+  width: 56px;
+  height: 40px;
+  font-weight: 500;
+  font-family: arial, sans-serif, 'Inter';
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  cursor: pointer;
+}
+
+.pro-btn .bs-btn {
+  border: 1px solid #8993a4;
+  width: 95px;
+  height: 40px;
+  font-weight: 500;
+  font-family: arial, sans-serif, 'Inter';
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  cursor: pointer;
+}
+
+.pro-btn .ruby-btn {
+  border: 1px solid #8993a4;
+  width: 61px;
+  height: 40px;
+  font-weight: 500;
+  font-family: arial, sans-serif, 'Inter';
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+  cursor: pointer;
+}
+
+.submit-pro-btn .see-pro {
+  width: 124px;
+  height: 48px;
+  background-color: #ff6b00;
+  gap: 10px;
+  border: none;
+  padding: 0;
+  cursor: pointer;
+}
```

Comment:
- [ ] Please 🙏🏽 add the correct space between the skills buttons and the see project button and correct the background color of the button it is #fff to be similar to the Figma template 👇🏽 

| Figma template | Your template |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/219762787-0794f2e8-f762-4b77-be15-143ddcbfdf56.png) | ![image](https://user-images.githubusercontent.com/58642949/219762930-0abadb7d-e642-43a0-86c9-a369d8e5864c.png) |